packages/plugin-core/src/_extension.ts

```diff
@@ -652,6 +654,9 @@ export async function _activate(
         workspaceType: ws.type,
         codeWorkspacePresent,
       });
+
+      warnConflictingExtensions();
```

Comment:
future optimization (create a next task, put in backlog and add a comment here, only run this check on first install and if the user has installed new extensions